app/services/nsm/assessment_syncer.rb

```diff
@@ -82,13 +82,14 @@ def sync_disbursements
 
     def sync_further_info_requests
       data['further_information'].each do |further_info|
-        claim.further_informations.find_or_create_by(
+        record = claim.further_informations.find_or_create_by(
```

Comment:
This bit looks redundant to me. The output of `find_or_create_by` will always be persisted to the database. `save` won't do anything because Rails knows nothing has changed.

Reply:
was thinking save would be needed for when we are setting the resubmission_deadline value but thinking about it, I guess because it's being set via a block from the find_or_create_by method I don't need to?

Reply:
Exactly. First it queries the database for an object matching the arguments. If it can't find one it writes to the database combining the arguments with whatever's set in the block. By the time the method has returned a value, that value is definitely in the DB.